docs/authoring-json-rules.md

```diff
@@ -22,12 +22,15 @@ Here are the fields that make up a rule definition.
 - The `helpUri` is optional, but it is good practice to include.  For built-in rules, this will point to a guide in the GitHub repository.
 
 ## Evaluation Object
-The evaluation object is comprised of three basic properties.
+The evaluation object is comprised of the following basic properties.
 ```javascript
 {
     "path": "JSON path to property to evaluate in template",
```

Comment:
As far as writing it succinctly, it can maybe be done in a table format or something.

If we remove the requirement, we'd also be able to write rules that check that a resource exists. Maybe we should discuss more about this and come back to this.

Reply:
Created issue #75 to track.